4.x info messages in 1.x

jdevcs opened the pull request:
## Description

https://github.com/ChainSafe/web3.js/issues/5411

Also removed broken badges

## Type of change

<!-- Please delete options that are not relevant. -->

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist for 1.x:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` with success.
- [ ] I have tested the built `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

## Checklist for 4.x:

- [ ] I have selected the correct 4.x base branch.
- [ ] Within the description, the feature or issue is discussed in detail or an issue is linked.   
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added any required tests for the changes I made 
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `yarn` successfully
- [ ] I ran `yarn lint` successfully
- [ ] I ran `yarn build:web` successfully
- [ ] I ran `yarn test:unit` successfully
- [ ] I ran `yarn test:integration` successfully
- [ ] I ran `compile:contracts` successfully
- [ ] I have tested my code.
- [ ] I have updated the corresponding `CHANGELOG.md` file in the packages I have edited.

jdevcs committed:
4.x message in readme

jdevcs committed:
4.x message in 1.x doc

jdevcs committed:
4.x message in post install message

avkos commented on packages/web3/README.md:
```diff
@@ -26,4 +28,6 @@ All the TypeScript typings are placed in the `types` folder.
 [deps-dev-image]: https://david-dm.org/ethereum/web3.js/1.x/dev-status.svg?path=packages/web3
-[deps-dev-url]: https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3
\ No newline at end of file
+[deps-dev-url]: https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3
```
this link returns 504 `https://david-dm.org/ethereum/web3.js/1.x?type=dev&path=packages/web3`

avkos reviewed (changes requested):
It will be great to fix not working links

jdevcs committed:
removed broken badges

jdevcs committed:
removed broken badge and added jsdelivr CDN badge

jdevcs commented:
> It will be great to fix not working links

removed , also added 
![image](https://user-images.githubusercontent.com/86780488/188620866-810dab19-1388-4af5-bacd-b4a588a4fd8b.png)

jdevcs committed:
Merge branch '1.x' into junaid/4.xInfoMessagesIn1.x

jdevcs committed:
Merge branch '1.x' into junaid/4.xInfoMessagesIn1.x

luu-alex commented on README.md:
```diff
@@ -4,7 +4,9 @@
-[![Discord][discord-image]][discord-url] [![StackExchange][stackexchange-image]][stackexchange-url] [![NPM Package Version][npm-image-version]][npm-url] [![NPM Package Downloads][npm-image-downloads]][npm-url] [![Build Status][actions-image]][actions-url] [![Dev Dependency Status][deps-dev-image]][deps-dev-url] [![Coverage Status][coveralls-image]][coveralls-url] [![Lerna][lerna-image]][lerna-url ...
+[![NPM Package Downloads][npm-image-downloads]][npm-url] [![cdnhits][cdnhits-image]][cdnhits-url] [![Discord][discord-image]][discord-url] [![StackExchange][stackexchange-image]][stackexchange-url] [![NPM Package Version][npm-image-version]][npm-url] [![Build Status][actions-image]][actions-url]  [![Coverage Status][coveralls-image]][coveralls-url] [![Lerna][lerna-image]][lerna-url] [![Netlify St ...
+
+#####  [Web3.js 4.x][4x-release] has been release. Checkout 4.x [API documentation and migration guide][4xdoc] for testing, early feedback and contributions. 
```
```suggestion
#####  [Web3.js 4.x][4x-release] has been released. Checkout 4.x [API documentation and migration guide][4xdoc] for testing, early feedback and contributions. 
```

luu-alex commented on docs/index.rst:
```diff
@@ -7,6 +7,8 @@ web3.js is a collection of libraries that allow you to interact with a local or
 
 The following documentation will guide you through :ref:`installing and running web3.js <adding-web3>` as well as providing an :ref:`API reference documentation <#id1>` with examples.
 
+.. note:: `Web3.js 4.x <https://github.com/ChainSafe/web3.js/releases/tag/v4.0.0-alpha.0>`_ has been release. Checkout 4.x `API documentation and migration guide <https://docs.web3js.org/>`_ for testing, early feedback and contributions. 
```
```suggestion
.. note:: `Web3.js 4.x <https://github.com/ChainSafe/web3.js/releases/tag/v4.0.0-alpha.0>`_ has been released. Checkout 4.x `API documentation and migration guide <https://docs.web3js.org/>`_ for testing, early feedback and contributions. 
```

luu-alex commented on packages/web3/README.md:
```diff
@@ -1,6 +1,8 @@
 # web3
 
-[![NPM Package][npm-image]][npm-url] [![Dependency Status][deps-image]][deps-url] [![Dev Dependency Status][deps-dev-image]][deps-dev-url]
+#####  [Web3.js 4.x][4x-release] has been release. Checkout 4.x [API documentation and migration guide][4xdoc] for testing, early feedback and contributions. 
```
```suggestion
#####  [Web3.js 4.x][4x-release] has been released. Checkout 4.x [API documentation and migration guide][4xdoc] for testing, early feedback and contributions. 
```

luu-alex commented:
just a small comment, otherwise lgtm

jdevcs committed:
Update docs/index.rst

Co-authored-by: Alex  <alex.luu@mail.utoronto.ca>

jdevcs committed:
Update packages/web3/README.md

Co-authored-by: Alex  <alex.luu@mail.utoronto.ca>

jdevcs committed:
Update README.md

Co-authored-by: Alex  <alex.luu@mail.utoronto.ca>
